test/framework/kubernetesversions/zz_generated.bindata.go

```diff
@@ -1,285 +0,0 @@
-/*
```

Comment:
was this removed intentionally? I don't see it being replaced with embed in this PR

Reply:
This was replaced by go embed in the previous PR, those are left-overs